errors.go

```diff
@@ -99,20 +99,30 @@ func (e *e) Cause() error {
 	return e.cause
 }
 
-// Wrap returns an error annotating the cause with message.
-// If cause is nil, Wrap returns nil.
-func Wrap(cause error, message string) error {
+func wrap(cause error, message string) error {
```

Comment:
Sorry, that's not what I meant, just copy Wrap to Wrapf and use fmt.Error
in place of errors.New. delete wrap

On Mon, 25 Apr 2016, 11:30 Umair Idris, <notifications@github.com> wrote:

> In errors.go <https://github.com/pkg/errors/pull/7#discussion_r60858190>:
>
> > @@ -99,20 +99,30 @@ func (e *e) Cause() error {
> >  	return e.cause
> >  }
> >
> > -// Wrap returns an error annotating the cause with message.
> > -// If cause is nil, Wrap returns nil.
> > -func Wrap(cause error, message string) error {
> > +func wrap(cause error, message string) error {
>
> Done.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/pkg/errors/pull/7/files/65849e46e9fbc498ba4a787faa474cd56ad87b45#r60858190>
>